geoportailv3/static/js/maincontroller.js

```diff
@@ -407,7 +407,9 @@ app.MainController.prototype.enable3dCallback_ = function(active) {
   if (!active) {
     return;
   }
-  this['mymapsOpen'] = false;
+  if (!goog.isDef(this.ngeoLocation_.getParam('map_id'))) {
```

Comment:
I think there exists an `hasParam` method in ngeo.
When the param exists it means we are not in edition?